### PR TITLE
Point hotkey

### PR DESCRIPTION
**Pull request by scrubmcnoob**
straight to the point
:cl:
Middle mouse button now points.
/:cl:

### CODE REVIEW

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -274,20 +274,20 @@
...
+/mob/proc/ShiftMiddleClickOn(atom/A)
 	return
 
 /mob/living/carbon/MiddleClickOn(atom/A)
 	if(!src.stat && src.mind && src.mind.changeling && src.mind.changeling.chosen_sting && (iscarbon(A)) && (A != src))
 		next_click = world.time + 5
 		mind.changeling.chosen_sting.try_to_sting(src, A)
 	else
-		swap_hand()
+		..()
```
Comment (scrubmcnoob) [inline code/_onclick/click.dm]: So you don't randomly swap hands while you're trying to use a ling sting cause that's not supposed to be the button's function.

File: code/_onclick/click.dm [hunk 2]
```diff
@@ -274,20 +274,20 @@
...
 
-/mob/living/simple_animal/drone/MiddleClickOn(atom/A)
+/mob/living/simple_animal/drone/ShiftMiddleClickOn(atom/A)
```
Comment (FlattestGuitar) [inline code/_onclick/click.dm]: This is only for drones. :v

File: code/_onclick/click.dm [hunk 3]
```diff
@@ -368,7 +368,7 @@
 	A.CtrlShiftClick(src)
 	return
 
-/mob/proc/ShiftMiddleClickOn(atom/A)
+/mob/proc/MiddleClickOn(atom/A)
```
Comment (FlattestGuitar) [inline code/_onclick/click.dm]: There are no checks at all here. Dead people can point.

File: code/_onclick/click.dm [hunk 4]
```diff
@@ -274,20 +274,20 @@
 	return
 
 /*
-	Middle click
+	Shift Middle click
```
Comment (FlattestGuitar) [inline code/_onclick/click.dm]: Shift click is not used for swapping hands in your implementation, from what I can tell.

File: code/_onclick/click.dm (hunk 3 above)
Comment (SamHPurp) [reply code/_onclick/click.dm]: To be fair, ghosts can point on both TG and Para.

Comment (scrubmcnoob) [reply code/_onclick/click.dm]: It's not broken. Tested it. Alive people can't see ghost points.

File: code/_onclick/click.dm (hunk 4 above)
Comment (scrubmcnoob) [reply code/_onclick/click.dm]: Fixed the note to correctly state what the function is now.

**Pull request merged**